Templates/RemoteRepo/Template/repo.json

```
@@ -0,0 +1,13 @@
+{
+    "repo_name":"${Name}",
+    "repo_uri": "${RepoURI}",
+    "repo_path": "${RepoPath}",
+    "schemaVersion": "1.0.0",
```

There should be a `$` character in the field name which was chosen to resemble the `$schema` field from the [official JSON schema docs](https://json-schema.org/understanding-json-schema/reference/schema.html#schema) 
```suggestion
    "$schemaVersion": "1.0.0",
```

---

Heads up, JSON uses $schema not $schemaVersion. I think keep it consistent. Either consistent with json and use $schema, or keep it consistent with O3D and use "version" or "schemaVersion" (no $)

---

$schemaVersion is already used throughout our python code